### ISSUE TITLE
Hunter / Pet - Pet Last Hitting Creatures

### ISSUE THREAD

**Issue by Cruzix** [dalaranwow/dalaran-wow#1048]
I experienced alot of times when pets are last hitting creatures the quest wont count and so on.

I confirmed this issue at this quest "Weakness to Lightning" in Borean Tundra. 

At first I thought the quest was bugged but then I searched in here if any issue thread was up, but since there was none I tried killing a few more creatures and then suddenly I got progress in the quest and realised I actually had to last hit, and when my pet was last hitting I wouldnt get any progress done.

(By last hitting I mean to be the one finishing the creature)

**Comment by DrFuler** [comment]
[04/03/2016] - The quest "Weakness to Lightning" is doable, but very buggy.

After killing 10+ robots while having my pet with me, I looked the quest up and came to the comments above. Indeed trying without the pet gave me a better luck (however, it also made this quest a pain in the ass). Here are some "tips" (not sure how true those are, but worked for me):

-> Don't use your pet. It has been your living shield for 60 levels, you can give it a one-quest vacation.
-> For some reason the Scavengers didn't seem to work even without the pet. My luck was a lot better with the spider-thingy ones.
-> I was under the impression that using the quest item before doing any damage gave me better odds at actually getting credit for it.

Took me 30+ robots to get the 15 credits I needed, and most of it was with no pet, so again, this bug is a pain in the ass (I haven't had a problem like that in any other quest, though).

**Comment by Rushor** [comment]
this was fixed together with:



DB/Quest:  The lost Mistwhisper Treasure - Always reward questcredit from Warlord Tartek

* special questevent from quest_template_addon is no longer needed
* moreover the credit will be rewarded to the whole group now
* also added some adjustments to SMART_ACTION_CALL_GROUPEVENTHAPPENS
* closes https://github.com/dalaranwow/dalaran-wow/issues/3917
* commit: 1fcdbcc265ac5cc078a009004e0eee51afb3e080

SMART_ACTION_CALL_AREAEXPLOREDOREVENTHAPPENS, SMART_ACTION_CALL_GROUPEVENTHAPPENS, SMART_ACTION_CALL_KILLEDMONSTER give the credit from the pet towards the player now.

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by Rushor** [comment]
The lost Mistwhisper Treasure was a bit modified in 428f48797a5317b84ab2e96d802dc585537baad2
